### PR TITLE
Remove Deadlock status

### PR DESCRIPTION
**Pull request by Atlante45**
This removes the recurring deadlock status log message.

### CODE REVIEW

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -278,17 +273,6 @@ class DeadlockWatchdogThread : public QThread {
                     << "maxElapsed:" << _maxElapsed
```
Comment (zzmp) [inline interface/src/Application.cpp]: There's a typo on the line above this, `VALUE**` instead of `VALUE **`.
It'd be nice to fix, but not necessary.

**Comment by zzmp** [comment]
What'd you change when you force-pushed?

**Comment by Atlante45** [comment]
@zzmp Forgot to remove the 2 `static const` at the top.

File: interface/src/Application.cpp (hunk 1 above)
Comment (Atlante45) [inline interface/src/Application.cpp]: OCD much? ;)
Will fix.

**Comment by hifi-gustavo** [comment]
Build console output for commit: cdb5af008524283a5d6ce88468fa50c604f72e09 available at: https://gist.github.com/6773d72804c5d3aca17a42978eee67f2<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7851.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7851.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: c4209c0a2dfff547073580feb492adfd56c67108 available at: https://gist.github.com/6773d72804c5d3aca17a42978eee67f2<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7851.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7851.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 5c196f3df5b6f57644df63a8b22984dfb6fae650 available at: https://gist.github.com/6773d72804c5d3aca17a42978eee67f2<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7851.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR7851.dmg

**Comment by kevinmthomas-carpool** [comment]
testing

**Comment by kevinmthomas-carpool** [comment]
## QA Test
**PR Build:** 7851
**Testing Architecture:** 64-bit PC Desktop
**Testing OS:** Windows 10
**Testing Locations:** localhost, music, playa, sandbox
**Additional Test Results/Observations:** After moving around multiple domains, only found 3 instances of the ```DEADLOCK``` warning.  Log included below.  
**Successful Logout/Crash:** Successful logout.
[hifi-log.txt](https://github.com/highfidelity/hifi/files/257044/hifi-log.txt)

**Pull request merged**